ceph.spec.in

```diff
@@ -1530,10 +1530,7 @@ fi
 %{_libdir}/librbd_tp.so.*
 %endif
 
-%post -n librbd1
-/sbin/ldconfig
-mkdir -p /usr/lib64/qemu/
-ln -sf %{_libdir}/librbd.so.1 /usr/lib64/qemu/librbd.so.1
+%post -n librbd1 -p /sbin/ldconfig
```

Comment:
Nit: should we be nice and remove the symlink upon upgrade if it exists?

Reply:
@dillaman Are we sure that qemu doesn't create such a symlink? I'm a little hesitant to reach into `/usr/lib64/qemu` and delete stuff . . .

Reply:
Fair enough -- I don't know enough history to know if this package is the only one that ever created that symlink